### DISCUSSION TITLE
Multiport Ethernet Switch Testing

### DISCUSSION THREAD

**Discussion by relaxibus** [chad-ermacora/rp-network-testers#16] [Ideas]
Could you add a function to use multiple RPi's so we can test a complete Ethernet switch?

**Comment by chad-ermacora** [comment]
Hmm, that's a good idea. Set a bunch of Pis as endpoints for testing one after the other. 

Yeah, I should be able to add that. I have not been programing for a bit, so I can't guarantee a timeframe, but yes, it's now on my todo list.

**Reply by relaxibus** [reply]
Good to hear that. May be possible to have Sequential and Spread/mesh (any to any) options?
